scripts/test

```diff
@@ -51,6 +51,20 @@ if [ ! -f "$vimdir/bin/vim" ]; then
   exit 1
 fi
 
+
+# Make sure all Go tools and other dependencies are installed.
+echo "Installing Go binaries"
+"$vimgodir/scripts/run-vim" $vim +':silent :GoUpdateBinaries' +':qa'
```

Comment:
This should be protected by an `if` so it doesn't get run on every test invocation.

Reply:
What kind of check do you propose?

Reply:
Maybe check the contents of GOPATH or something?

Right now running tests is very fast, which is a nice feature to have IMHO. This would make it significantly slower.

(I also think that install-vim is a better place btw, but practicality beats
purity, so if this makes solving the Docker problem a lot easier then this is
okay).